dev/examples/controls/bar.js

```diff
@@ -275,7 +275,7 @@ export const renderBarTimeSeriesWithDateline = (id) => {
             label: {
                 display: "Action Date"
             },
-            color: "#FFDF00",
+            color: "#C97318",
```

Comment:
Is this color is present in the carbon color constants? If yes, can we use it from constants in the below files?

Reply:
No it is not present in constants.

Reply:
Is it worth it to move the color constants to carbon color constants? Is this only used for dev site? or used by consumer also?

Reply:
@Veera81 It is not worth to move those to color constants because those are only used in dev site.